### ISSUE TITLE
Netlify Hosting

### ISSUE THREAD

**Issue by elie222** [zeit/next.js#3928]
Can Next easily be hosted on Netlify?

I have seen some mention of it in old issues from around a year ago, but wasn't fully clear if they work well together yet or not.

I'm assuming this is similar to hosting on GitHub pages, so I would hope the answer for Netlify is also a yes, but doesn't seem to be too popular an option for some reason and don't see much about it when doing a Google search.

So in short, is it possible to host Next on Netlify?
Do you lose out on any features by doing so?

**Comment by sergiodxa** [comment]
AFAIK Netlify is only for static sites, so it should be possible if you use `next export` to create an static site as with GitHub Pages.

**Comment by timneutkens** [comment]
Exactly what @sergiodxa is saying 🙏

**Comment by majdi** [comment]
Hello,

I'm trying to host my next app on Netlify but I have some issues with `next export`
I think it's due to es6 I have
```js
...
(function (exports, require, module, __filename, __dirname) { import _ from 'lodash'
                                                              ^^^^^^
SyntaxError: Unexpected token import
...
```

I really don't want to make some ugly hack with require :D

Thanks,

**Comment by timneutkens** [comment]
*never* put `"env"` in .babelrc when using Next, it's part of next/babel.

**Comment by majdi** [comment]
Ok thanks for that, I just remove it but same issue :)

**Comment by timneutkens** [comment]
If you can provide a minimal reproduction I can look into it 👍

**Comment by majdi** [comment]
Hello, you can check a simple repo here: https://github.com/majdi/next.js-issue.git

Actually, I need the "env" in my .babelrc to run the `yarn dev` command.
You can see that `yarn build` or `yarn export` fail because of *import* things.

Thanks :)

**Comment by tonyxiao** [comment]
How has this changed since netlify now supports lambda functions out of the box?

**Comment by majdi** [comment]
Hi, I didn't retry and host with now but I think it can works now :)
[ I removed my testing repository ]
